src/app/add-course-page/add-course-page/course-authors/course-authors.component.ts

```diff
@@ -48,32 +71,51 @@ export class CourseAuthorsComponent implements OnInit, ControlValueAccessor {
   registerOnTouched(fn: any): void {
     this.onTouched = fn;
   }
+
   onFocus(): void {
     this.showAuthorsList = true;
   }
 
   onBlur(): void {
     this.onTouched();
+    setTimeout(() => this.showAuthorsList = false, 300)
   }
 
   onAuthorNameClick(author: IAuthors): void {
-    this.authorNameClick.emit(author);
 
+    let currentAuthorList: IAuthors[] = this.pickedAuthors.value;
+
+    if (this.pickedAuthors.value[0] === null) {
+      this.pickedAuthors.removeAt(0);
+      currentAuthorList = this.pickedAuthors.value;
+    }
+
+    if (currentAuthorList.find((el: IAuthors) => el.id === author.id)) {
```

Comment:
so as I got right here you check if user clicked on already existing author you do nothing?
if yes - that's wrong, first of all it's strange behavior for user, because he clicking but nothing happening, secondly - you shouldn't show author which already picked at all